ambry-api/src/main/java/com/github/ambry/config/ServerConfig.java

```diff
@@ -112,6 +112,12 @@ public class ServerConfig {
   @Default("com.github.ambry.server.AmbryServerSecurityServiceFactory")
   public final String serverSecurityServiceFactory;
 
+  /**
+   * The information of the RepairRequestsDB to connect with.
+   */
+  @Config("server.repair.requests.db")
+  public final String dbInfo;
```

Comment:
we should have a factory configuration here and create different implementations for RepairRequestsDB, one with mysql database, one with inmemory data structure.

Reply:
Added the factory